README.md

```diff
@@ -41,7 +41,10 @@ dune build -p dot-merlin-reader,merlin
 
 Note: if you want to work on Merlin, you'll want to avoid the `-p merlin` to
 build in dev mode, with some extra warnings enabled. In that case, you'll also
-need an extra dependency: Menhir.
+need an extra dependency: Menhir. You should pin it by running:
```

Comment:
"We recommend that you pin it to version `20201216` which is the version of Menhir that was used to generate the parser  that is currently present in the sources.